NoClassDefFoundError: com/ccadllc/cedi/dtrace/LoggingFactory$

Looks like `LoggingFactory` should be in `com/ccadllc/cedi/dtrace/logging`. Fix in #59.

```
root[ERROR] java.lang.NoClassDefFoundError: com/ccadllc/cedi/dtrace/LoggingFactory$
root[ERROR] 	at com.ccadllc.cedi.dtrace.logging.LoggingConfig$.$anonfun$initialize$1(LoggingConfig.scala:33)
root[ERROR] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
root[ERROR] 	at cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:87)
root[ERROR] 	at cats.effect.internals.IORunLoop$.start(IORunLoop.scala:34)
root[ERROR] 	at cats.effect.internals.IOBracket$.$anonfun$apply$1(IOBracket.scala:44)
root[ERROR] 	at cats.effect.internals.IOBracket$.$anonfun$apply$1$adapted(IOBracket.scala:34)
root[ERROR] 	at cats.effect.internals.IORunLoop$RestartCallback.start(IORunLoop.scala:337)
root[ERROR] 	at cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:119)
root[ERROR] 	at cats.effect.internals.IORunLoop$RestartCallback.signal(IORunLoop.scala:351)
root[ERROR] 	at cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:372)
root[ERROR] 	at cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:312)
root[ERROR] 	at cats.effect.internals.IOShift$Tick.run(IOShift.scala:36)
root[ERROR] 	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
root[ERROR] 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
root[ERROR] 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
root[ERROR] 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
root[ERROR] 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
root[ERROR] Caused by: java.lang.ClassNotFoundException: com.ccadllc.cedi.dtrace.LoggingFactory$
root[ERROR] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
root[ERROR] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
root[ERROR] 	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
root[ERROR] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
root[ERROR] 	... 17 more
```

--- a/logging/js/src/main/scala/com/ccadllc/cedi/dtrace/logging/LoggingFactory.scala
+++ b/logging/js/src/main/scala/com/ccadllc/cedi/dtrace/logging/LoggingFactory.scala
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 package com.ccadllc.cedi.dtrace
+package logging
 
 import slogging.{ UnderlyingLoggerFactory, WinstonLoggerFactory }
 

--- a/logging/jvm/src/main/scala/com/ccadllc/cedi/dtrace/logging/LoggingFactory.scala
+++ b/logging/jvm/src/main/scala/com/ccadllc/cedi/dtrace/logging/LoggingFactory.scala
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 package com.ccadllc.cedi.dtrace
+package logging
 
 import slogging.{ SLF4JLoggerFactory, UnderlyingLoggerFactory }
 

